application/server/server_smoke_test.go

```diff
@@ -523,9 +619,12 @@ func setupRepoAndInitialize(t *testing.T, repo string, commit string, loc server
 	return cloneTargetDir
 }
 
-func checkFeatureFlagStatus(t *testing.T, loc *server.Local, c *config.Config) {
+func checkFeatureFlagStatus(t *testing.T, c *config.Config, loc *server.Local) {
```

Comment:
when we pass a config, it should be the first param (or second if it's a test helper)